mopro-ffi/src/app_config/android.rs

```diff
@@ -32,10 +30,35 @@ pub fn build() {
 
     install_ndk();
     for arch in target_archs {
-        build_for_arch(&arch, &build_dir, &bindings_out, &mode);
+        build_for_arch(arch, &build_dir, &bindings_out, &mode);
     }
 
-    write_kotlin_bindings(&bindings_out);
+    let uniffi_bindgen_path = {
+        let mut bin_path = std::env::current_exe().unwrap(); //Users/~/mopro/target/debug/ios
+        bin_path.pop(); // Remove the current executable name: //Users/~/mopro/target/debug
+        bin_path.pop(); //Users/~/mopro/target/
+        bin_path.pop(); //Users/~/mopro
+        bin_path.push("mopro-ffi");
+        bin_path
+    };
```

Comment:
I am thinking to move the `uniffi-bindgen` to `test-e2e` and other apps that build bindings
(though `mopro-ffi` will not generate bindings anymore 👀)

or in this case
when I want to use `mopro` cli to generate templates for mobile apps
I have to 
1. install the whole `mopro` repo 
2. fix `uniffi-bindgen` path 
    ( I cannot easily use `mopro init` in other directories, only `test-e2e` works)

How do you think?

Reply:
I don't think I understand. Do you mean that this way of composing the relative path wouldn't work for a general case?

Reply:
I don't understand what you mean by "fixing the uniffi-bindgen path".

Reply:
>  Do you mean that this way of composing the relative path wouldn't work for a general case?

Yes
We will need to clone the `mopro` repo and use the `uniffi-bindgen` [here](https://github.com/zkmopro/mopro/blob/61f6b364bf9052a8d83251a7a4104b9ff541dede/mopro-ffi/Cargo.toml#L16)

I imagine the user can download the `mopro-cli` binary (e.g. like a circom [binary](https://github.com/iden3/circom/releases/tag/v2.2.1))
and execute `mopro init` in any directory
and with the `mopro init`, there will be a `uniffi-bindgen` in the new directory

Reply:
I think I understand the problem now. It would be solved if there was a way to call `uniffi::uniffi_bindgen_main()` directly with arguments.